qutebrowser/completion/completiondelegate.py

```diff
@@ -196,13 +196,10 @@ def _get_textdoc(self, index):
 
         if index.parent().isValid():
             pattern = index.model().pattern
-            if index.column() == 0 and pattern:
-                repl = r'<span class="highlight">\g<0></span>'
-                text = re.sub(re.escape(pattern), repl, self._opt.text,
-                              flags=re.IGNORECASE)
-                self._doc.setHtml(text)
-            else:
-                self._doc.setPlainText(self._opt.text)
+            repl = r'<span class="highlight">\g<0></span>'
+            text = re.sub(re.escape(pattern), repl, self._opt.text,
+                          flags=re.IGNORECASE)
+            self._doc.setHtml(text)
```

Comment:
I haven't tried yet, but I could imagine this highlighting being annoying for e.g. command completion... Maybe the completion model should be able to decide which models it wants highlighted (e.g. via a class-level constant which is used here)?